go.mod

```diff
@@ -3,167 +3,109 @@ module github.com/filecoin-project/motion
 go 1.20
 
 require (
+	github.com/data-preservation-programs/singularity v0.3.1-0.20230908045144-b48f89bbe21e
```

Comment:
we may need v0.4.1 because it requires a bug fix from a PR i have open and 0.4.0 has already been merged (`ListAttachedWallets` swagger client returns `models.ModelWallet` when it should return `[]models.ModelWallet`)